test/models/accountSet.ts

```diff
@@ -0,0 +1,161 @@
+import { ValidationError } from 'ripple-api/common/errors'
+import { verifyAccountSet } from './../../src/models/transactions/accountSet'
+import { assert } from 'chai'
+
+/**
+ * AccountSet Transaction Verification Testing
+ *
+ * Providing runtime verification testing for each specific transaction type
+ */
+describe('AccountSet Transaction Verification', function () {
+
+    it (`verifies valid AccountSet`, () => {
+        const account = {
+            TransactionType : "AccountSet",
+            Account : "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            Fee : "12",
+            Sequence : 5,
+            Domain : "6578616D706C652E636F6D",
+            SetFlag : 5,
+            MessageKey : "03AB40A0490F9B7ED8DF29D246BF2D6269820A0EE7742ACDD457BEA7C7D0931EDB"
+        } as any
+            
+        assert.doesNotThrow(() => verifyAccountSet(account))
+    })
+
+    it (`throws w/ invalid SetFlag`, () => {
+        const offer = {
+            TransactionType : "AccountSet",
+            Account : "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+            Fee : "12",
+            Sequence : 5,
+            Domain : "6578616D706C652E636F6D",
+            SetFlag : 12,
+            MessageKey : "03AB40A0490F9B7ED8DF29D246BF2D6269820A0EE7742ACDD457BEA7C7D0931EDB"
+        } as any
+
+        assert.throws(
+            () => verifyAccountSet(offer),
+            ValidationError,
+            "AccountSet: invalid SetFlag"
+        )
+    })
+
+    it (`throws w/ invalid SetFlag`, () => {
```

Comment:
Can you make the description a bit more specific since this is a different case than the above test with the same name?

Reply:
Done!